service/src/main/kotlin/fi/espoo/evaka/pis/controllers/FosterParentController.kt

```diff
@@ -121,13 +128,18 @@ class FosterParentController(private val accessControl: AccessControl) {
                         Action.FosterParent.UPDATE,
                         id,
                     )
-                    tx.updateFosterParentRelationshipValidity(id, validDuring)
+                    tx.updateFosterParentRelationshipValidity(
+                        id,
+                        validDuring,
+                        clock.now(),
+                        Modifier.User(user.evakaUserId),
+                    )
                 }
             }
             .also {
                 Audit.FosterParentUpdateRelationship.log(
                     targetId = AuditId(id),
-                    meta = mapOf("validDuring" to validDuring),
+                    meta = mapOf("validDuring" to validDuring, "modifiedBy" to user.evakaUserId),
```

Comment:
as commented by Joonas in the other PR user is automatically passed to audit log